pipreqs/mapping

```diff
@@ -1,6 +1,7 @@
 AG_fft_tools:agpy
 ANSI:pexpect
 Adafruit:Adafruit_Libraries
+apache-airflow:airflow
```

Comment:
I believe you reversed it, it should be import_name:pypi_name, like so:
airflow:apache-airflow